eiffel-vocabulary/EiffelArtifactCreatedEvent.md

```diff
@@ -70,44 +56,16 @@ EXACTLY_ONE signifies that a legal composition must contain one and only one imp
 AT_LEAST_ONE signifies that a legal composition must contain one or more implementations of this artifact.
 
 ### data.implements
-__Type:__ Object[]  
-__Required:__ No  
-__Description:__ An array of [GAVs](https://maven.apache.org/guides/mini/guide-naming-conventions.html) this artifact implements. The typical use case of this is to identify interfaces implemented by this artifact.
-
-#### data.implements.groupId
 __Type:__ String  
-__Required:__ Yes  
-__Description:__ The groupId of the implemented artifact.
-
-#### data.implements.artifactId
-__Type:__ String  
-__Required:__ Yes  
-__Description:__ The artifactId of the implemented artifact.
-
-#### data.implements.version
-__Type:__ String  
-__Required:__ Yes  
-__Description:__ The version of the implemented artifact. Note that [version range notation](https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402) is supported.
-
-### data.dependsOn
-__Type:__ Object[]  
+__Format:__ [purl specification](https://github.com/package-url/purl-spec)  
 __Required:__ No  
-__Description:__ An array of [GAVs](https://maven.apache.org/guides/mini/guide-naming-conventions.html) this artifact depends on.
-
-#### data.dependsOn.groupId
-__Type:__ String  
-__Required:__ Yes  
-__Description:__ The groupId of the dependency.
+__Description:__ An array of [purl identified](https://github.com/package-url/purl-spec) entities this artifact implements. The typical use case of this is to identify interfaces implemented by this artifact. Note that [version range notation](https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402) may be used for the version component of the package identity.
```

Comment:
There seems to be no version range syntax defined in purl but what it states is that the version field should be percent-encoded. I guess we should then complement the Note in this section with that the given version range needs to be percent-encoded.

Reply:
Right, I attempted to clarify this in the most recent commit.